src/main/java/com/napier/sem/countries/Country.java

```diff
@@ -0,0 +1,50 @@
+package com.napier.sem.countries;
+
+import com.napier.sem.SqlManager;
+
+public class Country {
+
+    SqlManager sqlManager;
+
+    public Country(SqlManager sqlManager){
+        this.sqlManager = sqlManager;
+    }
+
+    public String worldCountries(){
+        String stmt = "SELECT name, code, continent, region, population, capital FROM country ORDER BY population DESC";
+
+        return sqlManager.executeStatement(stmt);
+    }
+
+    public String continentCountries(){
+        String stmt = "SELECT name, code, continent, region, population, capital FROM country WHERE continent = 'Africa' ORDER BY population DESC";
+
+        return sqlManager.executeStatement(stmt);
+    }
+
+    public String regionCountries(){
+        String stmt = "SELECT name, code, continent, region, population, capital FROM country WHERE region = 'Middle East' ORDER BY population DESC";
+
+        return sqlManager.executeStatement(stmt);
+    }
+
+    public String topNPopulatedCountries(int n){
+        String stmt = "SELECT name, code, continent, region, population, capital FROM country ORDER BY population DESC LIMIT <nHolder>";
```

Comment:
Have a look at string interpolation instead of doing `.replace()` here https://stackoverflow.com/questions/6389827/string-variable-interpolation-java